### ISSUE TITLE
Notable doesn't notice filename changes if a note has been tagged manually

### ISSUE THREAD

**Issue by wAuner** [notable/notable#467]
- **OS Version**: macOS 10.14.3
- **Notable Version**: 1.3.0

### Current behavior
I've exported all my notes from notion and imported them into notable. The exported files all had some cryptic name extensions like "note1-x9032-f89032.md".

On import into notable all the notes have been tagged with __Import-aa6c__.

I then wrote a script to clean up the filenames. While all filenames were changed successfully, I noticed that notable only updated the titles of the files, which I had not already tagged manually as well.

Example:
two files are imported and automatically tagged:
* file1_abc10101.md
* file2_nfjkds840932.md

Then file1 is tagged with my custom tag.
Then the filenames are changed.
The new filenames are `file1.md` and `file2.md` . 

Now notable shows the titles:
* `file1_abc10101`
* `file2`

So it didn't update the title of the note that has already been tagged.

### Expected behavior

Update the title consistently.

**Comment by fabiospampinato** [comment]
@wAuner Can you post your script? It seems weird that tags had any effect on this. Also, were you updating the file name or the title in the metadata? Because the latter is what Notable is using.

I think we should have an option for refreshing all titles 🤔

**Comment by wAuner** [comment]
hey @fabiospampinato, it's simple python script which did nothing more than renaming the files. not any different from doing it manually. That's what surprised me.

I noticed that when I first imported my notes and didn't do anything with it, the only metadata that's added is the tags field with the import tag.

At what point does the title field get added in the metadata? I suspect that when I tag a note manually, that that's when the title field gets added. This could explain the behavior.

**Comment by fabiospampinato** [comment]
> At what point does the title field get added in the metadata? I suspect that when I tag a note manually, that that's when the title field gets added. This could explain the behavior.

When a note is saved on this. If at that point it didn't have a `title` metadata one is created for it.

**Comment by wAuner** [comment]
but how about imports? I have tons of notes imported and most of them only have the `tags` field.

**Comment by fabiospampinato** [comment]
Yeah maybe this should run after importing the notes 🤔

**Comment by wAuner** [comment]
So the reason is probably:
I imported the notes and the few notes where I added manual tags before renaming, also received the `title` field in their metadata.
Once I renamed all the files, the files which I previously tagged (and thus received the `title` field) still display the old name from before the renaming. And all the files which I didn't tag manually don't have that `title` field in their metadata. So the app falls back to using the filename.
